lib/client/Client.js

```diff
@@ -356,6 +356,32 @@ export default class Client {
   }
 
   /**
+   * Destroy multiple {@link Mailbox}es specified to the given _ids_ on the server.<br />
```

Comment:
There is `These IDs must be in the right order...` and the API consumer can check http://jmap.io/spec.html#destroying-mailboxes to know there reason